.changeset/clever-bottles-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openproject/octicons': minor
+---
+
+make it possible to have xsmall icon
```

Comment:
Since this is the text for the release notes, please clarify that a bit. `xsmall` is already possible in rails, it is only added for angular